src/mocks/data/draft_reports_object.json

```diff
@@ -0,0 +1,50 @@
+{
+  "status": "processing", // enum: one of ["processing", "failed", "delivered"]
+  "current_stage": "dex-upload",
+  "current_action": "metadata-validation",
+  "file_name": "test.txt",
+  "transport_id": "unique_guid", //represents the id for the entire transaction i.e. upload_id
+  "timestamp": "12-12-2023", -- first initial timestamp
+  "data_stream_id": "aims-celr",
+  "data_stream_route": "csv",
+  "reports": [
+    {
+    "stage": "DEX HL7 v2", //describes a chunk of the pipeline, usually associated with a specific product; string could be human readable, or transformed from id to human readable
+    "action": "debatch", //describes specific thing that happened as part of the stage
+    "schema_name": "dex-hl7",
+    "version": "1.0",
```

Comment:
should read schema_version, to be precise there's other versions (like process_version or stage_version)

Reply:
Agreed, updated.